setup.cfg

```diff
@@ -1,35 +1,35 @@
 [metadata]
 name = pybasic
 version = 0.0.1
-author = Lorenz Lamm, Tingying Peng, Mohammad Mirkazemi
-author_email = tingying.peng@helmholtz-muenchen.de, Lorenz.lamm@helmholtz-muenchen.de
 description = A python package for background and shading correction of optical microscopy images
 long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/peng-lab/PyBaSiC
```

Comment:
Change url

Reply:
Also, add yourself as an author now that you've contributed :)